### PR TITLE
Make location-type code read-only

### PR DESCRIPTION
**Pull request by esoergel**
Location Type code will save once, on initial creation, then can't be modified again.  The only place it is visible is in the restore.
@sravfeyn

### CODE REVIEW

File: corehq/apps/locations/management/commands/check_loc_types.py [hunk 1]
```diff
@@ -1,157 +0,0 @@
-import csv
```
Comment (esoergel) [inline corehq/apps/locations/management/commands/check_loc_types.py]: This file relies on code a bunch, and isn't really relevant any more

**Comment by esoergel** [comment]
@sravfeyn ping

**Comment by czue** [comment]
(not mergeable)

**Comment by sravfeyn** [comment]
Looks good. Feel free to self-merge after resolving conflict.

**Pull request merged**